docs/reference/api-conventions.asciidoc

```diff
@@ -302,6 +302,14 @@ Content-Type: application/vnd.elasticsearch+json; compatible-with=7
 Accept: application/vnd.elasticsearch+json; compatible-with=7
 ----------------------------------------------------------------------
 
+[discrete]
+[[api-push-back]]
+=== `TOO_MANY_REQUESTS (429)` status code push back
+
+Elasticsearch may respond with a `TOO_MANY_REQUESTS (429)` status code, indicating that it is too busy right now to
+handle the request. When this happens, you should consider to retry the operation after a short delay, preferably
```

Comment:
```suggestion
handle the request. When this happens, you should retry the operation after a short delay, preferably
```

Reply:
I included "consider to" on purpose, since in some cases (UI for instance), a retry may not make sense.

Reply:
Ah right. 'consider to retry' is a bit of an odd phrase, 'you should consider retrying the operation' would be better